index.js

```diff
@@ -28,23 +28,26 @@ module.exports = function installGit(options) {
 
         var GIT_TEMPLATE_DIR = path.join(targetDirectory, 'usr/share/git-core/templates');
         var GIT_EXEC_PATH = path.join(targetDirectory, 'usr/libexec/git-core');
+        var LD_LIBRARY_PATH = path.join(targetDirectory, 'usr/lib64');
         var binPath = path.join(targetDirectory, 'usr/bin');
 
         function done() {
             if (updateEnv) {
                 process.env.PATH = process.env.PATH + ":" + binPath;
                 process.env.GIT_TEMPLATE_DIR = GIT_TEMPLATE_DIR;
                 process.env.GIT_EXEC_PATH = GIT_EXEC_PATH;
+                process.env.LD_LIBRARY_PATH = LD_LIBRARY_PATH;
```

Comment:
It's possible somebody might already have LD_LIBRARY_PATH set, since it has wider usage, so this could break things. Can you change this to append it to an existing value with a `:`, if it's already set?

I _think_ we should only do that here when directly updating the env, not when just returning the variables to use below, but I'm open to other opinions there.

Reply:
sure doing it now :)